### PR TITLE
[12.0] [MIG] partner_contact_department

### PR DESCRIPTION
**Pull request by Nikul-Chaudhary**
Partner Contact Department
==========================
This module extends the functionality of the address book to support departments.

Department is a drop-down field in partner forms, and it refers to contact department in its own company.

### CODE REVIEW

**Comment by Nikul-Chaudhary** [comment]
@pedrobaeza Can you plese help here https://travis-ci.org/OCA/partner-contact/jobs/482522564#L1242

Why Travis are failing?

**Comment by pedrobaeza** [comment]
Look for the error on logs:
```
2019-01-21 18:08:55,595 6442 ERROR openerp_test odoo.models: add a field parent_path on model res.partner.department: parent_path = fields.Char(index=True)
```

**Comment by Nikul-Chaudhary** [comment]
@pedrobaeza Done

**Comment by pedrobaeza** [comment]
Please squash together adjacent translation files.

**Review by pedrobaeza — APPROVED** [review]

**Comment by Nikul-Chaudhary** [comment]
@pedrobaeza squashed

**Comment by pedrobaeza** [comment]
I still see several adjacent "Update partner_contact_department.pot" and "README.rst" that can be squashed together.

**Comment by Nikul-Chaudhary** [comment]
> I still see several adjacent "Update partner_contact_department.pot" and "README.rst" that can be squashed together.

@pedrobaeza squashed

**Comment by pedrobaeza** [comment]
OK, thanks.

File: partner_contact_department/models/res_partner.py [hunk 1]
```diff
@@ -22,17 +23,23 @@ class ResPartnerDepartment(models.Model):
...
+        ondelete='restrict'
+    )
     child_ids = fields.One2many(
         "res.partner.department",
         "parent_id",
-        "Child departments")
+        "Child departments"
+    )
     parent_left = fields.Integer(index=True)
     parent_right = fields.Integer(index=True)
```
Comment (MiquelRForgeFlow) [inline partner_contact_department/models/res_partner.py]: delete `parent_left` and `parent_right`

and in the model, line 21, change `_order = "parent_left"` by `_order = "parent_path"`

and also add a simple post-migration script like https://github.com/OCA/partner-contact/pull/710/commits/f58d83931ac525a255f9a80ee951485d1b65dbe1#diff-497a43b26bdee7dabca3bf51c451683d

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

**Comment by Nikul-Chaudhary** [comment]
@mreficent Done, Review Again

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM :+1:

**Review by NL66278 — APPROVED** [review]
:+1:  LGTM

**Pull request merged**